src/components/modal/v2/parts/ContentWrapper.jsx

```diff
@@ -19,11 +19,13 @@ const ContentWrapper = () => {
         product = useServerData().views[0].meta.product;
     }
 
-    const { headline, subheadline } = useContent(product);
+    const { headline, subheadline, qualifyingSubheadline } = useContent(product);
```

Comment:
```suggestion
    const { headline, subheadline, qualifyingSubheadline = '' } = useContent(product);
```
Let's set a default here just to be safe since other views don't have ```qualifyingSubheadline``` in their content.